File: PlatziCodingChallenge_Dia14/NumerosPrimos.py
# Números primos en Python
def isPrime(number):
    for divider in range(2, int(number**.5) + 1):
        if(number % divider == 0):
            return False
    return True

def primeNumbers(limit):
    primes = []
    for number in range(2, limit + 1):
        if isPrime(number): primes.append(number)
    return primes

#Asking for the limit
n = int(input("¿Hasta qué número quiere saber los primos? "))
while(n < 2):
    print('Debe poner un número mayor o igual que 2')
    n = int(input("¿Hasta qué número quiere saber los primos? "))

#Printing the result
print(primeNumbers(n))

File: Dia14/NumerosPrimos.js
//Números primos en JS

function isPrime(number){
    for (let divider = 2; divider < Math.floor(Math.sqrt(number)) + 1; divider++) {
        if (number % divider == 0) {
            return false
        }
    }
    return true
}

function primeNumbers(limit){
    let primes = []
    for (let number = 2; number <= limit; number++) {
        if (isPrime(number)) {
            primes.push(number)
        }
    }
    return primes
}

do {
    var limite = parseInt(prompt('¿Hasta qué número quiere saber los primos?'))
    if (limite < 2) alert('Debe introducir un número igual o mayor que 2')
} while (limite < 2)

alert(`Números primos menores que ${limite}: ${primeNumbers(limite)}`)